nodejs/eks/nodegroup.ts

```diff
@@ -1203,15 +1203,18 @@ ${customUserData}
         : {};
 
     const device = pulumi.output(amiId).apply((id) =>
-        aws.ec2.getAmi({
-            owners: ["self", "amazon"],
-            filters: [
-                {
-                    name: "image-id",
-                    values: [id],
-                },
-            ],
-        }, { parent }),
+        aws.ec2.getAmi(
+            {
+                owners: ["self", "amazon"],
+                filters: [
+                    {
+                        name: "image-id",
+                        values: [id],
+                    },
+                ],
+            },
+            { parent },
+        ),
```

Comment:
Nit: formatting
```suggestion
        aws.ec2.getAmi({
           owners: ["self", "amazon"],
           filters: [
                {
                    name: "image-id",
                    values: [id],
                },
            ],
        }, { parent }),
```

This is just easier for me to read.